src/components/postgresConnect.ts

```diff
@@ -664,8 +664,12 @@ class PostgresDB {
     }
   }
 
-  public getDistinctPermissions = async (isSystemOrgPermission: boolean = false) => {
-    const text = 'select * from permission_name where is_system_org_permission = $1'
+  public getTemplatePermissions = async (isSystemOrgPermission: boolean = false) => {
+    const text = `SELECT template.code as "templateCode", pn.name as "permissionName", description 
+      FROM permission_name pn
+      JOIN template_permission tp ON pn.id = tp.permission_name_id
+      join template on tp.template_id = template.id
+      WHERE is_system_org_permission = $1`
```

Comment:
You probably don't need these joins, we already have them joined in the "permissions_all" view (see the queries above)

Reply:
Yes... but you don't have it filtered by `is_system_org_permission` there - and it would be more work to add, so I created a separated one

Reply:
I think we should add `is_system_org_permission` to the view then, since this View is meant to be a one-stop-shop for getting all permission-related stuff

Reply:
ok - I'll add this first thing after lunch

Reply:
Done f59c429